api/src/zero/module/aou.clj

```diff
@@ -30,15 +30,6 @@
   (let [[key value] (first cromwell-label-map)]
     (str (name key) ":" value)))
 
-(def chip-metadata
```

Comment:
I'm not up to date on the discussion on this. So long as we really don't need this anymore even as a default this all looks good.